plugins/blip/basics/less/misc/DailyDoseTitle.less

```diff
@@ -1,10 +1,16 @@
 .DailyDoseTitle {
+
   span.DailyDoseTitle-label {
     padding-right: 35px;
   }
   span.DailyDoseTitle-value {
     color: @text;
     font-weight: bold;
     font-size: @huge-numbers;
+    padding-left: 5px;
+  }
+  .icon-right:before {
+    margin-left: -15px;
+    margin-right: 0px;
```

Comment:
Norgie is misaligned when Avg total daily dose section is opened:
<img width="329" alt="screenshot 2016-12-05 10 24 01" src="https://cloud.githubusercontent.com/assets/1588547/20897074/6443b2b4-bad5-11e6-9301-45d09cbbfcc6.png">